experiments/pages/RevisionInspector/RevisionInspector.xml

```diff
@@ -0,0 +1,546 @@
+<skuid__page unsavedchangeswarning="yes" personalizationmode="server" showsidebar="false" showheader="false">
+	<models>
+		<model id="Revisions" limit="7" query="true" createrowifnonefound="false" datasource="salesforce" sobject="skuid__Page_Version__c" orderby="LastModifiedDate DESC">
+			<fields>
+				<field id="Name"/>
+				<field id="skuid__Layout__c"/>
+				<field id="skuid__Layout2__c"/>
+				<field id="skuid__Layout3__c"/>
+				<field id="skuid__Layout4__c"/>
+				<field id="skuid__Layout5__c"/>
+				<field id="CreatedById"/>
+				<field id="CreatedBy.Name"/>
+				<field id="CreatedDate"/>
+				<field id="skuid__Page__c"/>
+				<field id="skuid__Page__r.Name"/>
+				<field id="skuid__IsAutoSave__c"/>
+				<field id="LastModifiedDate"/>
+				<field id="skuid__Description__c"/>
+				<field id="Id"/>
+			</fields>
+			<conditions>
+				<condition type="param" value="id" field="skuid__Page__c" fieldtargetobjects="skuid__Page__c" clientorserver="server" state="filterableon" inactive="false" name="skuid__Page__c" operator="=" mergefield="Id" novaluebehavior="noquery" enclosevalueinquotes="true"/>
```

Comment:
```suggestion
				<condition type="param" value="id" field="skuid__Page__c" fieldtargetobjects="skuid__Page__c" clientorserver="server" state="filterableon" inactive="false" name="skuid__Page__c" operator="=" mergefield="Id" novaluebehavior="noquery" enclosevalueinquotes="true"/>
```

Not a necessary change, but if you set the condition to "Filterable default off" then the user can preview with the revision inspector without having to decide which page to inspect before opening it. That way they open the page and are prompted to search for the one they want—and the components don't appear until they select one.